tests/DetectsChangesTest.php

```diff
@@ -795,6 +795,46 @@ public function it_will_store_no_changes_when_wildcard_guard_and_log_unguarded_a
         $this->assertEquals([], $this->getLastActivity()->changes()->toArray());
     }
 
+    /** @test */
+    public function it_can_use_db_defaults_as_loggable_attributes()
+    {
+        $userClass = new class() extends User {
+            protected $fillable = ['name', 'text', 'currency'];
+            protected static $logAttributes = ['name', 'text', 'currency'];
+            protected static $logOnlyDirty = false;
+            protected static $logAttributesToIgnore = ['created_at', 'updated_at', 'deleted_at'];
+            use LogsActivity;
+        };
+
+        $user = new $userClass();
+        $user->name = 'my name';
+        $user->text = 'my text';
+        $user->save();
+        $expectedChanges = [
+            'attributes' => [
+                'name' => 'my name',
+                'text' => 'my text',
+                'currency' => 'USD',
+            ],
+        ];
+        $this->assertEquals($expectedChanges, $this->getLastActivity()->changes()->toArray());
+        $user->name = 'my name 1';
```

Comment:
I would like to also change the `currency` to `EUR` to check if it ignores the DB default value in this case.